compiler/ballerina-compiler-api/src/main/java/org/ballerina/compiler/impl/symbols/BallerinaModule.java

```diff
@@ -71,9 +74,10 @@ protected BallerinaModule(String name, PackageID moduleID, BPackageSymbol packag
         List<FunctionSymbol> functions = new ArrayList<>();
         for (Map.Entry<Name, ScopeEntry> entry : this.packageSymbol.scope.entries.entrySet()) {
             ScopeEntry scopeEntry = entry.getValue();
-            if (scopeEntry.symbol instanceof BInvokableSymbol &&
-                    (scopeEntry.symbol.flags & Flags.PUBLIC) == Flags.PUBLIC &&
-                    scopeEntry.symbol.kind != org.ballerinalang.model.symbols.SymbolKind.ERROR_CONSTRUCTOR) {
+            if (scopeEntry.symbol != null
+                    && scopeEntry.symbol.kind == org.ballerinalang.model.symbols.SymbolKind.FUNCTION
```

Comment:
Any specific reason to use FQCN `org.ballerinalang.model.symbols.SymbolKind.FUNCTION` here? or else shall we just use `SymbolKind.FUNCTION` for better readability

Reply:
There's `SymbolKind` class in the API package too. So had to use the FQ name to refer to the one in the lang module